commands/up.go

```diff
@@ -82,10 +84,31 @@ func preRunUp(cmd *cobra.Command, args []string) error {
 
 func upHandler(cmd *cobra.Command, args []string) error {
 	handler := upRunner(cmd, args)
+
+	// Always run an initial build to freshen up
 	if err := handler(); err != nil {
 		return err
 	}
 
+	var services stack.Services
+	if len(yamlFile) > 0 {
+		parsedServices, err := stack.ParseYAMLFile(yamlFile, regex, filter, envsubst)
+		if err != nil {
+			return err
+		}
+
+		if parsedServices != nil {
+			services = *parsedServices
+		}
+	}
+
+	fnNames := []string{}
```

Comment:
As, we need to replicate the capability into `local-run`. Possibly, we can wrap the entire watch code block and perform watch on the given list of functions.

Reply:
One note about local-run is that it only supports one function at a time right now.

Reply:
Yup. `fnNames` can be an array which can be used to filter functions which need to be watched.